lib/cachex.ex

```diff
@@ -302,6 +302,7 @@ defmodule Cachex do
          {:ok, pid} = Supervisor.start_link(__MODULE__, cache, name: name),
          {:ok, link} = Informant.link(cache),
          ^link <- Overseer.update(name, link),
+         _ = warm(cache, respect_async_warmers: true),
```

Comment:
First instinct is already that this looks a lot better. Minor nitpick but would prefer `Cachex.warm` rather than `warm` to make it more obvious we're re-using the public API.